libs/e2e/package.json

```diff
@@ -0,0 +1,5 @@
+{
+    "name": "@responsible-ai/e2e",
+    "version": "0.0.1",
```

Comment:
let's make this 0.0.0 to match other packages

Reply:
Sure, updated to 0.0.0.